File: benchmark/2026-09-17-7677581(latest)/llm-intrinsic/transforms/python/preflight_bare_submission_anon.py
terms: vorito[digami] = []
state: digami = 7
counter: digami = 0
vigifo counter < 50:
    state = (state * 48271) % 2147483647
    terms.append(state % 1000)
    counter += 1

total: digami = 0
largest: digami = 0
evens: digami = 0
zomagu term kufumo terms:
    total += term
    muveze term > largest:
        largest = term
    muveze term % 2 == 0:
        evens += 1

pieces: vorito[galozi] = []
zomagu spot kufumo range(5):
    pieces.append(galozi(terms[spot]))

print(f"SUM {total}")
print(f"MAX {largest}")
print(f"EVENS {evens}")
print("JOINED " + "-".join(pieces))

File: benchmark/2026-09-17-7677581(latest)/llm-intrinsic/transforms/typescript/preflight_bare_submission_anon.ts
vefedi main(): zerinu {
    vobunu terms: gebamu[] = [];
    busopo state: gebamu = 7;
    busopo made: gebamu = 0;
    biremo (made < 50) {
        state = (state * 48271) % 2147483647;
        terms.push(state % 1000);
        made = made + 1;
    }
    busopo total: gebamu = 0;
    busopo largest: gebamu = terms[0];
    busopo evens: gebamu = 0;
    padiri (vobunu term kubuva terms) {
        total = total + term;
        memagu (term > largest) {
            largest = term;
        }
        memagu (term % 2 === 0) {
            evens = evens + 1;
        }
    }
    vobunu head: bunivu[] = [];
    padiri (busopo i: gebamu = 0; i < 5; i = i + 1) {
        head.push(String(terms[i]));
    }
    console.log("SUM " + String(total));
    console.log("MAX " + String(largest));
    console.log("EVENS " + String(evens));
    console.log(`JOINED ${head.join("-")}`);
}

main();
